app/views/streams/main_stream.html.haml

```diff
@@ -44,20 +38,104 @@
+              = render "shared/invitations"
+
+        .section
+          .title
+            %h5.title-header
+              = t('aspects.index.new_here.title')
+          .content
+            != t('aspects.index.new_here.follow', link: link_to("#"+t('shared.publisher.new_user_prefill.newhere'), tag_path(name: t('shared.publisher.new_user_prefill.newhere'))))
+            %br
+            = link_to(t('aspects.index.new_here.learn_more'), "http://wiki.diasporafoundation.org/Welcoming_Committee")
+
+        .section
+          .title
+            %h5.title-header
+              = t('aspects.index.help.need_help')
+          .content
+            %p
+              = t('aspects.index.help.here_to_help')
+            %p
+              = t('aspects.index.help.do_you')
+            %ul
+              %li
+                != t('aspects.index.help.have_a_question', :link => link_to("#"+t('aspects.index.help.tag_question'), tag_path(:name => t('aspects.index.help.tag_question'))))
+              %li
+                != t('aspects.index.help.find_a_bug', :link => link_to("#"+t('aspects.index.help.tag_bug'), tag_path(:name => t('aspects.index.help.tag_bug'))))
+              %li
+                != t('aspects.index.help.feature_suggestion', :link => link_to("#"+t('aspects.index.help.tag_feature'), tag_path(:name => t('aspects.index.help.tag_feature'))))
+            %p
+              != t('aspects.index.help.tutorials_and_wiki',
+                   :faq => link_to(t('_help'), help_path),
+                   :tutorial => link_to(t('aspects.index.help.tutorial_link_text'), "https://diasporafoundation.org/tutorials", :target => '_blank'),
+                   :wiki => link_to('Wiki','http://wiki.diasporafoundation.org', :target => '_blank'), :target => '_blank')
+
+        - unless AppConfig.configured_services.blank? || all_services_connected?
+          .section
+            .title
+              %h5.title-header
+                = t('aspects.index.services.heading')
+            .content
+              %div
+                = t('aspects.index.services.content')
 
-    #right-sidebar-fixed-sub-background.col-lg-3.visible-lg
-    #right-sidebar-fixed-background.col-md-3.col-lg-2.hidden-sm.hidden-xs
-    .col-md-3.col-lg-2.hidden-xs.hidden-sm.rightbar
-      #selected_aspect_contacts.section
-        .title
-          %h5.stream_title
-            = @stream.title
-        .content
+              #right_service_icons
+                - AppConfig.configured_services.each do |service|
+                  - if AppConfig.show_service?(service, current_user)
+                    - unless current_user.services.any?{|x| x.provider == service}
+                      = link_to(content_tag(:div, nil, :class => "social_media_logos-#{service.to_s.downcase}-24x24", :title => service.to_s.titleize), "/auth/#{service}")
 
-      = render 'shared/right_sections'
+        .section
+          .title
+            %h5.title-header
+              = t('bookmarklet.heading')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.